blackbox/docs/config/secure.rst

```diff
@@ -0,0 +1,73 @@
+.. highlight:: sh
+
+.. _conf-secure-settings:
+
+===============
+Secure Settings
+===============
+
+Some settings are sensitive as they might store passwords, access keys or 
+other similar information. This type of data needs more protection than just 
+filesystem permissions and even more, should not be available in plain text.
+
+For these instances, we provide a keystore and the ``crate-keystore`` tool to 
+manage the settings in the keystore.
+
+.. NOTE::
+
+    Changes to the keystore take affect only after restarting CrateDB
+
+.. NOTE::
+
+    The CrateDB keystore currently provides obfuscation, but in the future 
+    password protection will be added.
```

Comment:
please clarify what this means. how does the obfuscation work?

Reply:
I think it's actually encryption so will change to that.